dear_petition/petition/api/viewsets.py

```diff
@@ -134,6 +156,21 @@ class TokenObtainPairCookieView(simplejwt_views.TokenObtainPairView):
 
     serializer_class = serializers.TokenObtainPairCookieSerializer
 
+    def get(self, request):
+        access_token = request.COOKIES.get(settings.AUTH_COOKIE_KEY)
+        if access_token is None:
+            return Response(status=status.HTTP_401_UNAUTHORIZED)
```

Comment:
I hit this line a lot running locally. Maybe we could add logging here to help debug, like:

```
logger.warning("Access token not found in cookie")
```

Reply:
Done